examples/fbcon/Kconfig

```diff
@@ -0,0 +1,201 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+config EXAMPLES_FBCON
+	tristate "Framebuffer console example"
+	default n
+	---help---
+		Enable the Framebuffer console example.
+		This example allows STDOUT and/or STDERR to be redirected and displayed
+		on a framebuffer character device.
+
+		A chosen builtin app can be spawned - such as "nsh" - to create a
+		console and, if required, STDIN can be intercepted to allow input
+		characters to be interpreted and pre-processed before passing them
+		on to the spawned app.
+
+		Relies on and uses NX and NXFONTS
+		
+if EXAMPLES_FBCON
+
+comment "Console STDIN and/or STDOUT setup"
+
+config EXAMPLES_FBCON_PIPE_STDOUT
+	bool "Pipe stdout to this console"
+	default y
+
+config EXAMPLES_FBCON_PIPE_STDERR
+	bool "Pipe stderr to this console"
+	default y
+
+config EXAMPLES_FBCON_PIPE_STDIN
+	bool "Pipe stdin via this console"
+	default y
+	---help---
+		This is usually nneded if the spawned App is "nsh" as we need to
+		intercept input characters in some cases
+
+comment "Console Framebuffer setup"
+
+config EXAMPLES_FBCON_DEF_FB
+	string "Default framebuffer driver"
+	default "/dev/fb0"
+	---help---
+	Default framebuffer drivers.  This selection can be overridden from
```

Comment:
add tab

Reply:
Thanks - had a VS Code moment as it lost all my language specific rules. Doh!
Will fix on next push